src/main/resources/application-tls.properties

```diff
@@ -0,0 +1,5 @@
+server.ssl.key-store-type=PKCS12
+server.ssl.key-store=file:${jdk.keystore.file}
+server.ssl.key-store-password=${jdk.keystore.password:changeit}
```

Comment:
If we can make the Helm chart autogenerate a password in a secret prior to deploy that'd be cool, but we should otherwise write it so that if jdk.keystore.password is undefined the app crashes rather than falling back to `changeit`?

Reply:
https://github.com/UKHomeOffice/cfssl-sidekick/blob/master/scripts/create-keystore.sh#L11 'changeit' is literally the password! it's not great is it. we could set up a better password in a secret and point this, and cfssl-sidekick at both, but I don't see an easy way for the sidekick to pick it up